tools/spec-gen-sdk/src/types/PackageData.ts

```diff
@@ -165,7 +165,7 @@ export const getPackageData = (context: WorkflowContext, result: PackageResult,
 
   // same logic as src/utils/utils.ts/removeDuplicatesFromRelatedFiles
   const typespecProjectIsManagementPlane = result.typespecProject && result.typespecProject.every(
-    item => item.endsWith('.Management')
+    item => (item.endsWith('.Management') || item.endsWith('resource-manager'))
```

Comment:
Can you update the second condition to check if the path contains 'resource-manager' because it doesn't exist in the end of the path?